github/challenge_processing_script.py

```diff
@@ -0,0 +1,103 @@
+import http
+import json
+import os
+import requests
+import sys
+
+from config import *
+from utils import (
+	add_pull_request_comment,
+	check_for_errors,
+	check_if_merge_or_commit,
+	check_if_pull_request,
+	create_challenge_zip_file,
+	create_github_repository_issue,
+	get_request_header,
+	load_host_configs,
+	validate_token,
+)
+
+sys.dont_write_bytecode = True
+
+GITHUB_CONTEXT = json.loads(os.getenv("GITHUB_CONTEXT"))
+
+GITHUB_AUTH_TOKEN = os.getenv("GITHUB_AUTH_TOKEN")
+if not GITHUB_AUTH_TOKEN:
+	print("Please add your github access token to the repository secrets with the name AUTH_TOKEN")
+	sys.exit(1)
+
+HOST_AUTH_TOKEN = None
+CHALLENGE_HOST_TEAM_PK = None
+EVALAI_HOST_URL = None
+
+
+if __name__ == "__main__":
+
+	configs = load_host_configs(HOST_CONFIG_FILE_PATH)
+	if configs:
+		HOST_AUTH_TOKEN = configs[0]
+		CHALLENGE_HOST_TEAM_PK = configs[1]
+		EVALAI_HOST_URL = configs[2]
+	else:
```

Comment:
Can we please add a print message here before exiting stating what might be the issue, something like HOSY_AUTH_TOKEN or CHALLENGE_HOST_TEAM_PK or EVALAI_HOST_URL is missing.

Reply:
It's done inside the util method which tries to load this data. Not needed here.